### PR TITLE
Add conditionals

### PR DESCRIPTION
**Pull request by maikia**
Hi @mkcor I very much used the SC material (excluding where the files are being loaded as we are not introducing this). I also kept most of the exercises, probably I won't have time to go through all of them, but it's nice to keep them for the students to do in their own time if they are motivated

### CODE REVIEW

File: _episodes/04-conditionals.md
Comment (mkcor) [inline _episodes/04-conditionals.md]:
```suggestion
title: "Making Choices with Conditionals"
```
Just for stylistic consistency, I prefer the longer (and gerund-based) syntax of episode titles from the [Programming with Python](https://swcarpentry.github.io/python-novice-inflammation/) SWC lesson vs [Plotting and Programming in Python](http://swcarpentry.github.io/python-novice-gapminder/index.html).

File: _episodes/04-conditionals.md
Comment (mkcor) [inline _episodes/04-conditionals.md:20]: I realize that question "What is a *statement*?" might come up; technically, the word appears twice in episode 00 and once in episode 01, so I've taken note of this.

File: _episodes/04-conditionals.md
Comment (mkcor) [inline _episodes/04-conditionals.md:44]: Personally I have replaced 'collection' with 'sequence' in the context of loops, because we only touch on examples where these collections happen to be sequences; maybe I should *say* that we can loop over collections which are not sequences (i.e., not ordered)... Also, but that's probably not a great argument, I prefer speaking of 'sequences' in this context, because (in a beginner setting) I use 'collections' when I say "a library is a *collection* of code units..."

File: _episodes/04-conditionals.md
Comment (mkcor) [inline _episodes/04-conditionals.md:111]: At this point, is it clear that 'conditional' is synonym to / short for 'conditional statement?' Reading this sentence, it sounds like a conditional consists of possibly several branches (so all {`if ...` , `elif ...` , `else`} would make up 'the' conditional), whereas reading the previous sections, it sounded as if each 'branch' would consist in a conditional (`if ...` is one conditional statement, `elif ...` is another one , etc.).

I believe that 'conditional' may refer to both, I'm just being the naive student here.

**Review by mkcor — APPROVED** [review]
Wonderful, @maikia! I've left a few comments. I'll merge #3 and maybe you can merge `gh-pages` into your branch before merging it. Thanks!

File: _episodes/04-conditionals.md
Comment (maikia) [reply _episodes/04-conditionals.md:111]: Do you propose that I make it as a single use and make it clear or that I explain it by saying it or that I prepare for questions about it?

Comment (maikia) [reply _episodes/04-conditionals.md:111]: I will merge and if you propose to still update it I will do it as a separate PR

Comment (mkcor) [reply _episodes/04-conditionals.md:111]: Right, you don't have to change the written materials, but just something to clarify and/or anticipate when you speak :+1: 

Thanks, @maikia!

**Pull request merged**
